<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>second</class>
 <widget class="QDialog" name="second">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>620</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog{
background-color:#595959;
}</string>
  </property>
  <widget class="QListWidget" name="ToDolistWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>120</y>
     <width>231</width>
     <height>281</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QListWidget{
background-color:#FFFFE2;
border-radius:50%;
}
QListWidget::item:focus{
outline:none;
}</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>80</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
font-family:&quot;Lucida Console&quot;, &quot;Courier New&quot;, monospace;
color:#7083E0;
}</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:16pt;&quot;&gt;To do List&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>80</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
color:#7083E0;
font-family:&quot;Lucida Console&quot;, &quot;Courier New&quot;, monospace;
}</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:16pt;&quot;&gt;Tasks&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ExitPushButton">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>570</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
font-family:&quot;Lucida Console&quot;, &quot;Courier New&quot;, monospace;
text-align: center;
border:1px solid;
color: black;
background-color:#969696;
}

QPushButton:hover{
background-color:#D4D4D4;
}</string>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>110</y>
     <width>111</width>
     <height>131</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPushButton" name="AddPushButton">
      <property name="styleSheet">
       <string notr="true">QPushButton{
border:none;
color:#318C99;
font-family:&quot;Lucida Console&quot;, &quot;Courier New&quot;, monospace;
font-size:20px;
}
QPushButton::hover{
text-decoration:underline;
color:#010203;
}</string>
      </property>
      <property name="text">
       <string>Add</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="EditPushButton">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
border:none;
color:#318C99;
font-size:20px;
font-family:&quot;Lucida Console&quot;, &quot;Courier New&quot;, monospace;
text-align: center;

}

QPushButton::hover{
text-decoration:underline;
color:#010203;
}</string>
      </property>
      <property name="text">
       <string>Edit</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="DeletePushButton">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
border:none;
color:#318C99;
font-family:&quot;Lucida Console&quot;, &quot;Courier New&quot;, monospace;
text-align: center;
font-size:20px;

}

QPushButton::hover{
text-decoration:underline;
color:#010203;
}</string>
      </property>
      <property name="text">
       <string>Delete</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QListWidget" name="InfoListWidget">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>340</y>
     <width>201</width>
     <height>141</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QListWidget{
background-color:#FFFFE2;
border-radius:50%;
}
QListWidget::item:focus{
outline:none;
}</string>
   </property>
  </widget>
  <widget class="QLabel" name="TaskDescriptionLabel">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>360</y>
     <width>131</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
color:#7083E0;
font-family:&quot;Lucida Console&quot;, &quot;Courier New&quot;, monospace;
}</string>
   </property>
   <property name="text">
    <string>Task Description:</string>
   </property>
  </widget>
  <widget class="QLabel" name="DeadLineLabel">
   <property name="geometry">
    <rect>
     <x>188</x>
     <y>450</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
color:#7083E0;
font-family:&quot;Lucida Console&quot;, &quot;Courier New&quot;, monospace;
}</string>
   </property>
   <property name="text">
    <string>DeadLine:</string>
   </property>
  </widget>
  <widget class="QLabel" name="InfoDeadLineLabel">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>450</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
color:#7083E0;
font-family:&quot;Lucida Console&quot;, &quot;Courier New&quot;, monospace;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="InfoTaskDescriptionLabel">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>360</y>
     <width>161</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
color:#7083E0;
font-family:&quot;Lucida Console&quot;, &quot;Courier New&quot;, monospace;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
